integration-testing/src/mavenTest/kotlin/MavenPublicationAtomicfuValidator.kt

```diff
@@ -33,20 +38,38 @@ class MavenPublicationAtomicfuValidator {
         val foundClasses = mutableListOf<String>()
         for (e in entries()) {
             if (!e.name.endsWith(".class")) continue
+            // check the bytecode of the class for ATOMIC_FU_REF excluding metadata
```

Comment:
Please drop a comment about when we can remove this workaround and why it is needed in the first place